dali/test/python/operator_2/test_random_bbox_crop.py

```diff
@@ -494,6 +510,64 @@ def get_boxes():
         pipe.run()
 
 
+def test_crop_window_warning():
+    n_boxes = [10, 12, 0]
+    pipe = Pipeline(batch_size=len(n_boxes), num_threads=1, device_id=0)
+
+    def get_boxes():
+        return [np.random.uniform(0, 1, size=(n, 4)).astype(np.float32) for n in n_boxes]
+
+    boxes = fn.external_source(source=get_boxes)
+    n_attempt = 10
+    processed = fn.random_bbox_crop(
+        boxes,
+        thresholds=[1.0],  # Impossible threshold to force the warning
+        bbox_layout="xyXY",
+        allow_no_crop=False,
+        total_num_attempts=n_attempt,
+    )
+    pipe.set_outputs(*processed)
+    with TemporaryFile(mode="w+") as f:
+        with redirect_stderr_fd(f.fileno()):
+            pipe.run()
+        f.seek(0)
+        logs = f.read().splitlines()
+    n_lines_expected = sum(n > 0 for n in n_boxes)
+    assert (
+        len(logs) == n_lines_expected
+    ), f"Expected {n_lines_expected} lines of output (one for each non-empty sample)"
+
+    expect_str = (
+        "[/workspaces/DALI/dali/operators/image/crop/bbox_crop.cc:838] Could not find a "
```

Comment:
I would not hardcode `[/workspaces/DALI/dali/operators/image/crop/bbox_crop.cc:838]` where the line number can change if the file is edited, same for the path that is true for your build but not necessarily for the release one.